webpages/css/main.css

```diff
@@ -198,17 +207,26 @@ main.index > section > * {
 }
 
 main.index > section:nth-of-type(4n+2) {
-  background-color: #90e8ef; /* from poster */
+  /*background-color: #90e8ef; /* from poster */
+  /*background-color: #D6EDFE; /* from poster */
+  background-image: linear-gradient(to bottom, #D6EDFE, #DADAFE);
+
   color: black;
 }
 
 main.index > section:nth-of-type(4n+3) {
-  background-color: #177894; /* from poster */
+  /*background-color: #177894; /* from poster */
+  /*background-color: #3992D6; /* from poster */
+  background-image: linear-gradient(to bottom, #0484E6, #3992D6);
```

Comment:
In meeting with Tomas, we discuss about Apple's website, where there is such gradients. So I was testing if these looks good or not and discuss these changes w/ the rest of the team, this is why also there is comment with previous versions.